src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+  .calendar {
+    flex-wrap: wrap;
+    gap: $calendar-gap;
+    width: calc(7 * $calendar-day-width + 7px);
+    list-style: none;
+    display: flex;
+
+    &__day {
+      cursor: pointer;
+      background-color: $calendar-day-color;
+      border: 1px solid black;
+      justify-content: center;
+      align-items: center;
+      height: $calendar-day-height;
+      width: $calendar-day-width;
+      transition: 500ms;
+      display: flex;
+      box-sizing: border-box;
+
+      &:hover {
+        background-color: #FFBFCB;
+        transform: translateY(-20px);
+
+      }
+
+      &::before {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: $calendar-day-font-size;
+        content: "";
+      }
+
+      @for $number from 1 through $calendar-length {
+        &:nth-child(#{$number})::before {
+          content: "#{$number}";
+        }
+      }
+    }
+
+    @each $day, $index in $days {
+      &--start-#{$day} > :first-child {
+        margin-left: ($index - 1) * ($calendar-day-width + $calendar-gap) ;
```

Comment:
Check the logic for calculating the `margin-left` for the first child. It should account for both the width of the days and the gaps between them. If `$calendar-gap` is defined as a gap between the days, the calculation should be `($index - 1) * ($calendar-day-width + 1) * $calendar-gap` assuming that the `$calendar-gap` is a multiplier of the gap size.